images/jupyter/Dockerfile.centos

```
@@ -4,7 +4,7 @@ FROM codercom/enterprise-base:centos
 USER root
 
 # Install jupyter
-RUN pip3 install jupyterlab
+RUN pip3 install jupyterlab && jupytercore-core==4.7.1
```

Looks like a typo, suggesting:

```suggestion
RUN pip3 install jupyterlab && jupyter-core==4.7.1
```

```sh
➜  ~ pip3 install jupytercore-core==4.8.1
ERROR: Could not find a version that satisfies the requirement jupytercore-core==4.8.1 (from versions: none)
ERROR: No matching distribution found for jupytercore-core==4.8.1
```